### PR TITLE
Pre Loader 

### PR DESCRIPTION
**Pull request by akshay-na**
# Changes

I have used the requested [Lottie](https://lottiefiles.com/58266-quad-cube-shifter-1) from the issue https://github.com/saadpasta/developerFolio/issues/357 with a slightly different color code.

## Main

- Modified Main.js to take Preloader component and render preloader component for 2 milliseconds at the beginning. 

## Components

- Added preloader compoenent.
- Modified DisplayLottie.js to accept height and width parameters to resize the Lottie.

## Assets

- Added preloader [Lottie](https://lottiefiles.com/58266-quad-cube-shifter-1).
- Added Pre.svg  ( a bar loader to display when illustration = false ).

### CODE REVIEW

**Comment by akshay-na** [comment]
This is my first ever contribution to an open source project. So please bear with me. 

Let me know if anymore changes/fixes are required

Also can I add a type writter effect below the name in the greetings page to describe the roles and positions? Like Developer, blogger etc etc.?

**Comment by akshay-na** [comment]
dark gif
light gif

**Comment by akshay-na** [comment]
dark
light

**Comment by saadpasta** [comment]
There is lot of indentation file changes please remove Thanks 🙏

File: src/components/preloader/Pre.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      }
+    >
+      {props.load ? (
+        illustration.animated ? (
+          <DisplayLottie animationData={PreloaderLottie} width={250} />
+        ) : (
+          <img
+            class="svg-loader "
+            alt="pre loader svg"
+            src={require("../../assets/pre.gif")}
```
Comment (kartikcho) [inline src/components/preloader/Pre.js:31]: Hey @akshay-na, sorry for the delayed response.
Why are we using a gif here as a Lottie fallback? 

Also #366 has already implemented the splashscreen loader (and it was opened before your PR so it wouldn't be nice to reject that piece of work )

Hate to reject your first PR on GitHub 😅 but this is why it's better to ask on an issue before making a PR for it so we can expect it.
Maybe if @ashutosh1297 is alright with you picking up their PR's latest review you can help ship their branch. Or you can check out issue #456 and take a stab at it

Comment (kartikcho) [reply src/components/preloader/Pre.js:31]: btw love your portfolio's loading animation ❤️

**Pull request closed without merging**